.ci.yaml

```diff
@@ -4403,6 +4403,17 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Windows windows_home_scroll_perf__timeline_summary
+    recipe: devicelab/devicelab_drone
+    presubmit: true
```

Comment:
The current workflow is using `bringup: true` first. 
We don't need `presubmit: true` here: 1) ci yaml roller use default value `true` for presubmit 2) With this set, it will cause the presubmit hang on this test as there is not available builder config yet.

Reply:
Done